### PR TITLE
Chinese translation

### PR DESCRIPTION
**Pull request by xinjiajuan**

### CODE REVIEW

File: packages/SystemUI/res-keyguard/values-zh-rCN/yaap_strings.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+** Unless required by applicable law or agreed to in writing, software
+** distributed under the License is distributed on an "AS IS" BASIS,
+** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+** See the License for the specific language governing permissions and
+** limitations under the License.
+*/
+-->
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+
+    <string name="clock_title_binary" translatable="false">二进制</string>
```
Comment (ALKALiKong233) [inline packages/SystemUI/res-keyguard/values-zh-rCN/yaap_strings.xml]: Well, I think we shouldn't translate the values that translatable is false

Comment (idoybh) [reply packages/SystemUI/res-keyguard/values-zh-rCN/yaap_strings.xml]: True

Comment (xinjiajuan) [reply packages/SystemUI/res-keyguard/values-zh-rCN/yaap_strings.xml]: > Well, I think we shouldn't translate the values that translatable is false

what？This is the values file itself

File: packages/SystemUI/res-keyguard/values-zh-rCN/strings.xml [hunk 2]
```diff
@@ -137,4 +137,108 @@
     <string name="clock_title_default" msgid="6342735240617459864">"默认"</string>
     <string name="clock_title_bubble" msgid="2204559396790593213">"泡泡"</string>
     <string name="clock_title_analog" msgid="8409262532900918273">"指针"</string>
+
+    <string name="clock_title_type" translatable="false">文字</string>
+
```
Comment (idoybh) [inline packages/SystemUI/res-keyguard/values-zh-rCN/strings.xml]:
```suggestion
```

File: packages/SystemUI/res-keyguard/values-zh-rCN/yaap_strings.xml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    <string name="clock_title_default_bold" translatable="false">默认 (粗体)</string>
+    <string name="clock_title_samsung" translatable="false">三星</string>
+    <string name="clock_title_samsung_bold" translatable="false">三星  (粗体)</string>
+
```
Comment (idoybh) [inline packages/SystemUI/res-keyguard/values-zh-rCN/yaap_strings.xml]:
```suggestion
```

**Review by idoybh — CHANGES_REQUESTED** [review]

**Review by ALKALiKong233 — APPROVED** [review]
Now it's ok after ido's commit. Remember never translate the values that has translatable false tag @ZONGQUAN-HU

File: packages/SystemUI/res-keyguard/values-zh-rCN/yaap_strings.xml (hunk 1 above)
Comment (idoybh) [reply packages/SystemUI/res-keyguard/values-zh-rCN/yaap_strings.xml]: see the `translatable="false"`? Never touch these.

**Comment by idoybh** [comment]
> Now it's ok after ido's commit. Remember never translate the values that has translatable false tag @ZONGQUAN-HU

Thanks for confirming. Will squash and merge

**Comment by xinjiajuan** [comment]
> Now it's ok after ido's commit. Remember never translate the values that has translatable false tag @ZONGQUAN-HU

OMG, I just saw it. I didn’t see this part without translation, I know, thank you

File: packages/SystemUI/res/values-zh-rCN/yaap_strings.xml [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    <string name="qs_menu_item_columns_five">5</string>
+    <string name="qs_menu_item_columns_six">6</string>
+    <string name="qs_menu_item_columns_seven">7</string>
+    <string name="qs_menu_item_columns_eight">8</string>
+    <string name="qs_menu_item_columns_nine">9</string>
+    <string name="qs_menu_item_reset">重置</string>
+    <string name="qs_menu_item_titles">显示磁贴标题</string>
+    <string name="qs_menu_item_qs_columns">快速栏磁贴个数</string>
+    <string name="qs_menu_item_columns_auto">自动</string>
+    <string name="qs_menu_item_titles">页脚文字</string>
```
Comment (idoybh) [inline packages/SystemUI/res/values-zh-rCN/yaap_strings.xml:100]: Which one of `qs_menu_item_titles` is right. We'll never know. I am removing the last one

Comment (poad42) [reply packages/SystemUI/res/values-zh-rCN/yaap_strings.xml:100]: Perhaps Google translate it?

Comment (xinjiajuan) [reply packages/SystemUI/res/values-zh-rCN/yaap_strings.xml:100]: yes，Did not find the same，You can delete the last line

**Pull request merged**